.github/terraform/google-cloud-build-triggers.tf

```diff
@@ -71,11 +87,11 @@ resource "google_cloudbuild_trigger" "custom-metrics-direct-to-sd" {
     }
 }
 
-resource "google_cloudbuild_trigger" "custom-metrics-prometheus-to-sd" {
-    name = "kubernetes-engine-samples-custom-metrics-prometheus-to-sd"
+resource "google_cloudbuild_trigger" "custom-metrics-prometheus-to-sd-gmp" {
+    name = "kubernetes-engine-samples-custom-metrics-prometheus-to-sd-gmp"
```

Comment:
a nit here and below (including the directories): `custom-metrics-gmp` 

We can remove the references to prometheus-to-sd altogether (including the naming) in favor of GMP.

Reply:
Done. (As discussed oob, I named the directory `google-managed-prometheus`, and the file `custom-metrics-gmp.yaml`.)